Cargo.toml

```diff
@@ -40,3 +40,9 @@ opt-level = 3
 [profile.release]
 lto = "thin"
 codegen-units = 1
+
+[profile.ci]
+inherits = "release"
```

Comment:
We might also want to set `codegen-units = 16` (or even higher, 16 is the default) in our CI profile to increase parallel compilation. Also apparently `lto = false` still performs "thin local LTO" (https://doc.rust-lang.org/cargo/reference/profiles.html#lto), so we could also experiment with `lto = "off"` as well.